packages/console/src/pages/SignInExperience/components/Welcome/GuideModal.tsx

```diff
@@ -39,15 +39,15 @@ const GuideModal = ({ isOpen, onClose }: Props) => {
   const {
     reset,
     handleSubmit,
-    formState: { isSubmitting, isDirty },
+    formState: { isSubmitting, isDirty, dirtyFields },
     watch,
   } = methods;
   const api = useApi();
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const formData = watch();
   const [isLoading, setIsLoading] = useState(false);
 
-  const previewConfigs = usePreviewConfigs(formData, isDirty, data);
+  const previewConfigs = usePreviewConfigs(formData, isDirty, dirtyFields, data);
```

Comment:
nit: almost meet the maximum attribute length limit for a method, convert it to an object prop?

Reply:
Refactored.